components/camel-http4/src/main/java/org/apache/camel/component/http4/HttpPollingConsumer.java

```diff
@@ -33,21 +33,33 @@
 import org.apache.http.client.methods.HttpGet;
 import org.apache.http.client.methods.HttpRequestBase;
 import org.apache.http.client.protocol.HttpClientContext;
+import org.apache.http.impl.auth.BasicScheme;
+import org.apache.http.protocol.HttpContext;
 import org.apache.http.util.EntityUtils;
 
 /**
  * A polling HTTP consumer which by default performs a GET
  *
  * @version 
  */
+@SuppressWarnings("deprecation")
```

Comment:
I would not prefer supressing deprecation. Especially while we have java's 6 month release cycle..